CONTRIBUTING.md

```diff
@@ -1,53 +1,51 @@
-# Reflex Contributing Guidelines
+# 🚀 Reflex Contributing Guidelines
 
-For an extensive guide on the different ways to contribute to Reflex see our [Contributing Guide on Notion](https://www.notion.so/reflex-dev/2107ab2bc166497db951b8d742748284?v=f0eaff78fa984b5ab15d204af58907d7).
+For a comprehensive guide on how to contribute to Reflex, check out our [Contributing Guide on Notion](https://www.notion.so/reflex-dev/2107ab2bc166497db951b8d742748284?v=f0eaff78fa984b5ab15d204af58907d7).
 
+## 🏃 Running a Local Build of Reflex
 
-
-## Running a Local Build of Reflex 
-Here is a quick guide to how the run Reflex repo locally so you can start contributing to the project.
+Here's a quick and fun guide to running the Reflex repository locally so you can start contributing to the project.
 
 **Prerequisites:**
-- Python >= 3.8
-- Poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
-
+- Python >= 3.8 🐍
+- Poetry version >= 1.4.0 📖 (add it to your path, see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
 
-**1. Clone Reflex and navigate into the repo:**
-``` bash
+**1. Clone Reflex and Navigate into the Repo:**
+```bash
 git clone https://github.com/reflex-dev/reflex.git
 cd reflex
 ```
 
-**2. Install your local Reflex build:**
-``` bash
+**2. Install Your Local Reflex Build:**
+```bash
 poetry install
 ```
-**3. Now create an examples folder so you can test the local Python build in this repository.**
+
+**3. Create an Examples Folder:**
 * We have the `examples` folder in the `.gitignore`, so your changes in `reflex/examples` won't be reflected in your commit.
-``` bash
+```bash
 mkdir examples
 cd examples
 ```
 
-**4. Init and Run**
-``` bash
+**4. Init and Run:**
+```bash
 poetry run reflex init
 poetry run reflex run
 ```
 All the changes you make to the repository will be reflected in your running app.
 
-
 ## 🧪 Testing and QA
 
-Within the 'test' directory of Reflex you can add to a test file already there or create a new test python file if it doesn't fit into the existing layout.
+In the 'test' directory of Reflex, you can add to an existing test file or create a new test Python file if it doesn't fit the current layout.
 
-#### What to unit test?
-- Any feature or significant change that has been added.
-- Any edge cases or potential problem areas.
-- Any interactions between different parts of the code.
+#### What to Unit Test?
+- Any new feature or significant change.
+- Edge cases or potential problem areas.
+- Interactions between different parts of the code.
 
-Now Init/Run
-``` bash
+Now, Init and Run:
```

Comment:
I think this part was repeated wrongly, should probably be removed as it already appear on line 33.